packages/containers/psammead-timestamp-container/src/index.stories.jsx

```diff
@@ -15,110 +15,161 @@ import '@bbc/psammead-locales/moment/ig';
 import '@bbc/psammead-locales/moment/pcm';
 import '@bbc/psammead-locales/moment/yo';
 
-let timestamp = moment();
+// Fixed timestamp for 27 August 2019, 14:54 BST
+const fixedTimestamp = 1566914061212;
 
-const date = name => {
-  button('now', () => {
-    timestamp = moment();
-  });
-  button('5 seconds ago', () => {
-    timestamp = moment().subtract(5, 'seconds');
-  });
-  button('30 seconds ago', () => {
-    timestamp = moment().subtract(30, 'seconds');
-  });
-  button('59 seconds ago', () => {
-    timestamp = moment().subtract(59, 'seconds');
-  });
-  button('1 minute ago', () => {
-    timestamp = moment().subtract(1, 'minute');
-  });
-  button('24 minutes ago', () => {
-    timestamp = moment().subtract(24, 'minutes');
-  });
-  button('59 minutes ago', () => {
-    timestamp = moment().subtract(59, 'minutes');
-  });
-  button('63 minutes ago', () => {
-    timestamp = moment().subtract(63, 'minutes');
-  });
-  button('95 minutes ago', () => {
-    timestamp = moment().subtract(95, 'minutes');
-  });
-  button('2 hours ago', () => {
-    timestamp = moment().subtract(2, 'hours');
-  });
-  button('23 hours ago', () => {
-    timestamp = moment().subtract(23, 'hours');
-  });
-  button('1 day ago', () => {
-    timestamp = moment().subtract(1, 'day');
-  });
-  button('30 hours ago', () => {
-    timestamp = moment().subtract(30, 'hours');
-  });
-  button('20 days ago', () => {
-    timestamp = moment().subtract(20, 'days');
-  });
-  button('21 days ago', () => {
-    timestamp = moment().subtract(21, 'days');
-  });
-  button('8.9999... months ago', () => {
-    timestamp = moment()
-      .subtract(9, 'months')
-      .add(2, 'second');
-  });
-  button('9 months ago', () => {
-    timestamp = moment().subtract(9, 'months');
+const val = momentTimestamp => momentTimestamp.valueOf();
+
+const timestamps = {
+  'Fixed (27 Aug 2019)': { timestamp: undefined, update: () => fixedTimestamp },
+  Now: { timestamp: undefined, update: () => Date.now() },
+  '5 seconds ago': {
```

Comment:
I cant see these anymore :(
![image](https://user-images.githubusercontent.com/11341355/64349064-2582b580-cfee-11e9-95e6-eab2fb50f9aa.png)

Reply:
@drew does it work for you if you put the `const`s back where they were? It doesn't seem to for me :/

Reply:
These are showing in storybook for me.

Reply:
@gabriellorin resolved the comment.